app/controllers/admin/banners_controller.rb

```diff
@@ -43,7 +43,10 @@ def create
   def update
     @admin_banner = AdminBanner.find(params[:id])
 
-    if @admin_banner.update_attributes(params[:admin_banner])
+    if @admin_banner.update_attributes(params[:admin_banner]) && params[:admin_banner_minor_edit]
```

Comment:
Actually, doing it this way you'll be updating_attributes twice if it is not a minor edit. I suggest you flip it, ie:

```
if !@admin_banner.update_attributes(params[:admin_banner])
  render action: 'edit'
elsif params[:admin_banner_minor_edit]
  minor edit code here
else
  normal success code here
end
```